tests/context.py

```diff
@@ -47,6 +47,9 @@ def get_elastic_ip(self, retry=3, interval=1):
 
         for serv in services.items:
             if serv.metadata.name == 'elasticsearch-master':
+                print("#@!#@! serv\n")
```

Comment:
what does this do? maybe add a comment to explain!

Reply:
this is a debugging print

Reply:
I don't think debug output belongs in production... is the plan to remove this before merging?

Reply:
removed